netcontrold/tests/lib/test_util.py

```diff
@@ -18,71 +18,170 @@
-            mock.mock_open(read_data=_BASIC_CPU_INFO_0).return_value
+            mock.mock_open(read_data=_BASIC_CPU_INFO_Processorid_NULL).return_value
         ]
         self.assertRaises(ValueError, util.cpuinfo)
+       
 
     @mock.patch('netcontrold.lib.util.open')
-    def test_cpuinfo_basic0(self, mock_open):
+    def test_cpuinfo_coreid(self, mock_open):
+        mock_open.side_effect = [
+            mock.mock_open(read_data=_BASIC_CPU_INFO_Processorid_NULL).return_value
+        ]
+        self.assertRaises(ValueError, util.cpuinfo)
+
+
+    @mock.patch('netcontrold.lib.util.open')
+    def test_cpuinfo_physical(self, mock_open):
+        mock_open.side_effect = [
+            mock.mock_open(read_data=_BASIC_CPU_INFO_Processorid_NULL).return_value
+        ]
+        self.assertRaises(ValueError, util.cpuinfo)
+        
+
+
+
+
+    
+
+    @mock.patch('netcontrold.lib.util.open')
+    def test_cpuinfo_basic(self, mock_open):
         mock_open.side_effect = [
-            mock.mock_open(read_data=_BASIC_CPU_INFO_1).return_value
+            mock.mock_open(read_data=_BASIC_CPU_INFO_Positive).return_value
         ]
         expected = [{'processor': '0', 'core id': '0', 'physical id': '0'},
-                    {'processor': '1', 'core id': '1', 'physical id': '0'},
-                    {'processor': '2', 'core id': '0', 'physical id': '0'},
+                    {'processor': '1', 'core id': '0', 'physical id': '0'},
+                    {'processor': '2', 'core id': '1', 'physical id': '0'},
                     {'processor': '3', 'core id': '1', 'physical id': '0'}]
         out = util.cpuinfo()
         self.assertEqual(out, expected)
 
     @mock.patch('netcontrold.lib.util.open')
-    def test_cpuinfo_empty(self, mock_open):
+    def test_cpuinfo_is_empty(self, mock_open):
         mock_open.side_effect = [
             mock.mock_open(read_data="").return_value
         ]
         out = util.cpuinfo()
         expected = []
         self.assertEqual(out, expected)
 
+
+
+
+
+
+
+class TestUtil_numa_cpu_map(TestCase):
+
+    @mock.patch('netcontrold.lib.util.open')
+    def test_numa_cpu_map_Negative(self, mock_open):
+        mock_open.side_effect = [
+            mock.mock_open(read_data=_BASIC_CPU_INFO_Negative).return_value
+        ]
+        self.assertRaises(ValueError, util.cpuinfo)
+
+
     @mock.patch('netcontrold.lib.util.open')
-    def test_numa_cpu_map(self, mock_open):
+    def test_numa_cpu_map_Processorid(self, mock_open):
         mock_open.side_effect = [
-            mock.mock_open(read_data=_BASIC_CPU_INFO_1).return_value
+            mock.mock_open(read_data=_BASIC_CPU_INFO_Processorid_NULL).return_value
+        ]
+        self.assertRaises(ValueError, util.cpuinfo)
+       
+
+    @mock.patch('netcontrold.lib.util.open')
+    def test_numa_cpu_map_coreid(self, mock_open):
+        mock_open.side_effect = [
+            mock.mock_open(read_data=_BASIC_CPU_INFO_Processorid_NULL).return_value
```

Comment:
I guess you mean mock.mock_open(read_data=_BASIC_CPU_INFO_Coreid_NULL).return_value